pkg/featureflags/featureflags_test.go

```diff
@@ -0,0 +1,17 @@
+package featureflags
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test FeatureFlags", func() {
+	Context("Testing OptionalFeatures", func() {
+		It("should initialize OptionalFeatures", func() {
+			optionalFeatures := OptionalFeatures{Backpressure: true}
+			Initialize(optionalFeatures)
```

Comment:
We can add one more test to verify that a second initialization does not change the feature flag (see `sync.Once`), since the feature flag pkg is meant to only initialize one time flags during controller start up.